README.md

```diff
@@ -58,6 +59,20 @@ Using [yarn](https://yarnpkg.com):
 yarn add @fingerprintjs/fingerprintjs-pro-react
 ```
 
+Using [pnpm](https://pnpm.js.org):
+
+```sh
+pnpm add @fingerprintjs/fingerprintjs-pro-react
+```
+
+## Requirements
+
+- React 18 or higher
+- For Preact users: Preact 10.3 or higher
+- For Next.js users: Next.js 13.1 or higher
+- For Typescript users: Typescript 4.8 or higher
+
+
```

Comment:
Include the FingerprintJS vs Pro section here: https://www.notion.so/fingerprintjs/Writing-DX-GitHub-Readmes-WIP-9556ed587c0749b29f32a18017413cbc?pvs=4#40df07e276694e3bba6c0c532690cdc2

Reply:
Done!